### DISCUSSION TITLE
How can auto installing bundles during installing pimcore?

### DISCUSSION THREAD

**Discussion by abdelrhmanyosry** [pimcore/pimcore#16502] [Q&A / Support] [answered]
I'm using some built-in bundles for pimcore in my company and we need to make it installed after every deployment how can we do it?

**Comment by AlephLau** [comment, marked as answer]
@abdelrhmanyosry 
You can Use Composer Post-Install Scripts
You can leverage Composer's post-install scripts to automate the installation of bundles.
In your composer.json file, add a post-install-cmd or post-update-cmd script that runs the necessary Symfony console commands to enable and install the bundles.
For example:
json
"scripts": {
    "post-install-cmd": [
        "@php bin/console pimcore:bundle:enable BundleName",
        "@php bin/console pimcore:bundle:install BundleName"
    ]
}

**Reply by abdelrhmanyosry** [reply]
thankyou for helping!
